pkg/apparmor/profiles/usr.bin.vtpm

```diff
@@ -1,14 +1,26 @@
-# Copyright (c) 2023 Zededa, Inc.
+# Copyright (c) 2024 Zededa, Inc.
```

Comment:
If this a brand new file (but reusing the old name?)

If is it a modification of the old file then you should keep the old year and add the new e.g., 2023,2024.

Reply:
Then 2024 is fine.